lib/internal/fs/sync.js

```diff
@@ -25,6 +25,18 @@ function readFileUtf8(path, flag) {
   return binding.readFileUtf8(path, stringToFlags(flag));
 }
 
+function writeFileUtf8(path, data, flag, mode) {
+  if (!isInt32(path)) {
+    path = pathModule.toNamespacedPath(getValidatedPath(path));
+  }
+
+  return binding.writeFileUtf8(
+    path, data,
```

Comment:
Can you move this function to lib/fs.js in accordance with @joyeecheung's pull request?